### ISSUE TITLE
206523: In Browse mode when Press "T" ,NVDA is announcing number of rows count incorrectly.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15133]
### Steps to reproduce:
Hit the URL:[row count and colcount bug (codepen.io)](https://codepen.io/aasimkhan30/pen/vYzLVYR) 
Tab till NVDA
Turn on Browse mode and press "T"
Verify NVDA is announcing number of rows count correctly or not
### Actual behavior:
In Browse mode when Press "T" ,NVDA is announcing number of rows count incorrectly

### Expected behavior:
In Browse mode when Press "T" ,NVDA is  should announce  rows count information correctly

#### NVDA version:
NVDA Version 2023.1

#### Windows version:
Edition Windows 11 Enterprise
Version 22H2
OS build 22621.1555
Experience Windows Feature Experience Pack 1000.22640.1000.0

**Comment by XLTechie** [comment]
The table claims 8619 rows. However, since it is growing dynamically as scrolled 
(in Brave), it is difficult to determine how many there actually are. How many 
exist in the JSON from which this is generated?

I find it interesting, @jcsteh, that NVDA does not recognize this as a navigable 
table in Firefox, but it does in Brave.
Is Chromium papering over some bad markup, to make NVDA see this as a table 
that Ctrl+Alt+arrows work in?

**Comment by seanbudd** [comment]
Could you please confirm what the expected row count is?
Could you please confirm what browser and what browser versions you can reproduce the bug with?

**Comment by jcsteh** [comment]
> I find it interesting, @jcsteh, that NVDA does not recognize this as a navigable
table in Firefox, but it does in Brave.

It does in Firefox Nightly for me. I'm guessing you were testing with Firefox 114 or 115? I landed a large refactor of some old table code in Firefox 116 (now beta). Among other things, that fixes tables containing intervening generic elements like this one.

**Comment by seanbudd** [comment]
This issue seems invalid and cannot be reproduced.
In future, please provide the browser and browser version you are reproducing this issue with.
